.github/workflows/athena_test.yml

```
@@ -64,25 +72,56 @@ jobs:
-          cd athena
-          # Set up Python path like test_modules.py does
-          export PYTHONPATH="$PWD:$PWD/llm_core"
           
-          # Run prospector only on the main athena directory to avoid import errors
-          poetry run prospector --profile .prospector.yaml athena/
-        continue-on-error: true
+          # Combine coverage data to avoid duplicates
+          echo "📊 Combining Coverage Data..."
+          echo "============================"
+          python3 combine_coverage.py test-results/combined_coverage.xml test-results/programming_module_programming_llm_coverage.xml test-results/text_module_text_llm_coverage.xml test-results/modeling_module_modeling_llm_coverage.xml
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)